t/091-coroutine.t

```diff
@@ -281,7 +281,7 @@ GET /lua
               end)
             end
 
-            N = 10 
+            N = 10
```

Comment:
Better avoid unrelated changes to make your changeset atomic. It's better for version control.